docs/docs/classic-ml/deep-learning/index.mdx

```diff
@@ -0,0 +1,401 @@
+- **A/B Testing** infrastructure for safe model rollouts
+
+<details>
+  <summary>Advanced Deployment Options</summary>
+
+  #### Beyond Basic Serving
+
+  MLflow supports sophisticated deployment scenarios for deep learning:
+
+  - **Model Ensembling**: Deploy multiple models with voting or averaging mechanisms
+  - **Custom Preprocessing/Postprocessing**: Attach data transformation pipelines to your model
+  - **Optimized Inference**: Support for quantization, pruning, and other optimization techniques
+  - **Monitoring Integration**: Connect to observability platforms for production tracking
+  - **Hardware Acceleration**: Leverage GPU/TPU resources for high-throughput inference
+  - **Scalable Architecture**: Handle variable loads with auto-scaling capabilities
+  - **Multi-Framework Deployment**: Mix models from different frameworks in the same serving environment
+</details>
+
+## Framework Integrations
+
+MLflow provides native support for all major deep learning frameworks, allowing you to use your preferred tools while gaining the benefits of unified experiment tracking and model management.
+
+<CardGroup>
+  <LogoCard description="Seamlessly track TensorFlow experiments with one-line autologging. Capture training metrics, model architecture, and TensorBoard visualizations in a centralized repository." link="/deep-learning/tensorflow">
+    <span>![TensorFlow Logo](/images/logos/TensorFlow-logo.svg)</span>
+  </LogoCard>
+
+  <LogoCard description="Integrate MLflow with PyTorch's flexible deep learning ecosystem. Log metrics from custom training loops, save model checkpoints, and simplify deployment for production." link="/deep-learning/pytorch">
+    <span>![PyTorch Logo](/images/logos/pytorch-logo.svg)</span>
+  </LogoCard>
+
+  <LogoCard description="Harness Keras 3.0's multi-backend capabilities with comprehensive MLflow tracking. Monitor training across TensorFlow, PyTorch, and JAX backends with consistent experiment management." link="/deep-learning/keras">
+    <span>![Keras Logo](/images/logos/keras-logo.svg)</span>
+  </LogoCard>
+
+  <LogoCard description="Track and manage spaCy NLP models throughout their lifecycle. Log training metrics, compare model versions, and deploy language processing pipelines to production." link="/model/#spacyspacy">
+    <span>![spaCy Logo](/images/logos/spacy-logo.svg)</span>
+  </LogoCard>
+
+  <LogoCard description="Streamline your work with Hugging Face Transformers. Track fine-tuning experiments, manage model versions, and deploy state-of-the-art language models with MLflow's specialized integration." link="/llms/transformers">
+    <span>![Hugging Face Logo](/images/logos/huggingface-logo.svg)</span>
+  </LogoCard>
+</CardGroup>
+
+## Getting Started
+
+<details>
+  <summary>Quick Setup Guide</summary>
+
+  ### 1. Install MLflow with Deep Learning Dependencies
+
+  ```bash
+  # For comprehensive support
+  pip install mlflow[extras]
```

Comment:
Do we need `[extras]`? I don't see related dependences used here in extras, also let's specify users should install tensorflow/pytorch themselves?